src/apify/_configuration.py

```diff
@@ -259,7 +259,15 @@ class Configuration(CrawleeConfiguration):
             description='TCP port for the Actor to start an HTTP server to receive messages in the Actor Standby mode',
         ),
         deprecated('Use `web_server_port` instead'),
-    ] = 4322
+    ] = 4321
+
+    standby_url: Annotated[
+        str,
```

Comment:
In Crawlee, we use a custom URL validator function:

```python
    url: Annotated[str, BeforeValidator(validate_http_url), Field(...)]
```

https://github.com/apify/crawlee-python/blob/v0.3.8/src/crawlee/_request.py#L112

@jirimoravcik Could we use the same approach here as well?

Reply:
sure